<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FullTextSearchDialog</class>
 <widget class="QDialog" name="FullTextSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Full text search</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="uiHelp">
       <property name="whatsThis">
        <string>Here you can search for any term in the whole database as long as it has been indexed. If the words you're searching don't don't return any results and you know they exist, they're probably not indexed and you should ask the system administrator.

Text search only searches for complete words but it will return words that contain the same root. For example, searching for &quot;build&quot;, may return documents which contain the words &quot;built&quot;, &quot;build&quot; or &quot;building&quot;.</string>
       </property>
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'helvetica'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;help&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;?&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="uiText"/>
     </item>
     <item>
      <widget class="QComboBox" name="uiModel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushFind">
       <property name="text">
        <string>&amp;Find</string>
       </property>
       <property name="icon">
        <iconset resource="common.qrc">
         <normaloff>:/images/find.png</normaloff>:/images/find.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWebView" name="uiWeb">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushPrevious">
       <property name="text">
        <string>&amp;Previous</string>
       </property>
       <property name="icon">
        <iconset resource="common.qrc">
         <normaloff>:/images/previous.png</normaloff>:/images/previous.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushNext">
       <property name="text">
        <string>&amp;Next</string>
       </property>
       <property name="icon">
        <iconset resource="common.qrc">
         <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushClose">
         <property name="text">
          <string>&amp;Close</string>
         </property>
         <property name="icon">
          <iconset resource="common.qrc">
           <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtWebKitWidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="common.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>uiText</sender>
   <signal>returnPressed()</signal>
   <receiver>pushFind</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
